src/node_contextify.cc

```diff
@@ -58,6 +58,9 @@ class ContextifyContext {
  public:
   ContextifyContext(Environment* env, Local<Object> sandbox_obj) : env_(env) {
     Local<Context> v8_context = CreateV8Context(env, sandbox_obj);
+    if (v8_context.IsEmpty())
```

Comment:
@bnoordhuis I won’t be able to take care of that until at least tomorrow… could you double-check that? I was actually getting a fatal error due to this, and I was surprised that the check below didn’t catch that and this one did.